### DISCUSSION TITLE
[IntelliSense] Auto class sorting

### DISCUSSION THREAD

**Discussion by hodev-dev** [tailwindlabs/tailwindcss#10286] [Ideas]
### Add auto class sorting feature
there is visual studio code extension for this but it doesn't support JIT or some features so it would be good if its part of main package
```
https://github.com/heybourn/headwind
```
thank for your great work

**Comment by yanggggjie** [comment]
Hi, there is an official prettier plugin for class sorting. https://tailwindcss.com/docs/editor-setup#automatic-class-sorting-with-prettier
